lib/fluent/plugin/output.rb

```diff
@@ -985,21 +987,28 @@ def try_flush
         end
 
         begin
+          chunk_write_start = Time.now
```

Comment:
Use `Process.clock_gettime(PROCESS_CLOCK_ID)` instead of `Time.now`.
`Time.now` is affected by system clock modification.